### ISSUE TITLE
Missing opening paragraph on CNN articles

### ISSUE THREAD

**Issue by leibovic** [mozilla/readability#281]
Articles on CNN frequently have an opening paragraph that is styled differently than others. Reader mode does not include this lead paragraph in its view.

See, for example, http://money.cnn.com/2016/02/01/news/economy/poverty-inequality-united-states/index.html

**Comment by gijsk** [comment]
This is because they use an `<h2>` for that text, with no classes or anything else, and Readability then does:

https://github.com/mozilla/readability/blob/master/Readability.js#L446-L449
```javascript
    // If there is only one h2, they are probably using it as a header
    // and not a subheader, so remove it since we already have a header.
    if (articleContent.getElementsByTagName('h2').length === 1)
      this._clean(articleContent, "h2");
```

I'm not really sure what to do about this... There doesn't seem to be any logic to how the h2 is being used - it really ought not to be a header in the first place. It's bad for accessibility and doesn't add anything to the structure of the page. They should just use a `<p>` and a class to style it how they want to. That said, I don't know that we'll be able to get them to fix that...

**Comment by evanxd** [comment]
Fixed by #347 .
